dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamFormatRestRepositoryIT.java

```diff
@@ -56,7 +57,16 @@ public class BitstreamFormatRestRepositoryIT extends AbstractControllerIntegrati
     @Autowired
     private BitstreamFormatConverter bitstreamFormatConverter;
 
-    private final int DEFAULT_AMOUNT_FORMATS = 85;
+    // Default number of formats in the registry (filled out in @Before method below)
```

Comment:
I know that it is boring fix this kind of stuff often but it is more.appropriate in my opinion an hardcoded check here. If we rely on our API to perform the check we loss the guarantee about which registries are loaded by default. This would hide unwanted changes.

Reply:
@abollini : I have to admit I get frustrated by hardcoded numbers like this, but I'll revert it as it's really unrelated to this PR.

Reply:
Reverted the change to make the count dynamic.  Now I just updated the hardcoded value: [8d81c14](https://github.com/DSpace/DSpace/pull/9638/commits/8d81c144bba435a5b515ae6e54cd65582f343eda)